usaspending_api/common/cache_decorator.py

```diff
@@ -19,14 +19,18 @@ def process_cache_response(self, view_instance, view_method, request, args, kwar
         if not response:
             response = view_method(view_instance, request, *args, **kwargs)
             response = view_instance.finalize_response(request, response, *args, **kwargs)
+            response['Cache-Trace'] = 'no-cache'
             response.render()  # should be rendered, before picklining while storing to cache
 
             if not response.status_code >= 400 or self.cache_errors:
                 try:
                     self.cache.set(key, response, self.timeout)
+                    response['Cache-Trace'] = 'set-cache'
                 except Exception as e:
                     msg = 'Problem while writing to cache: path:\'{p}\' data:\'{d}\''
                     logger.exception(msg.format(p=str(request.path), d=str(request.data)))
+        else:
+            response['Cache-Trace'] = 'hit-cache'
```

Comment:
Since its added to the response, it can be logged in the `server.log` and captured in Kibana. The field needs to be added to the logging config in `settings.py`. Then the `logging.py` file needs to be updated in `process_request` function.